docs/content/docs/integrations/aws.md

```diff
@@ -49,7 +49,7 @@ For example, to use AWS features with Spark 3 and AWS clients version 2.15.40, y
 ```sh
 # add Iceberg dependency
 ICEBERG_VERSION={{% icebergVersion %}}
-DEPENDENCIES="org.apache.iceberg:iceberg-spark3-runtime:$ICEBERG_VERSION"
+DEPENDENCIES="org.apache.iceberg:iceberg-spark-runtime-3.2_2.12:$ICEBERG_VERSION"
```

Comment:
@jackye1995 I wasn't sure if AWS is on 3.2 spark yet

Reply:
Actually not yet. I think we can just change L47 to say `to use AWS features with Spark 3.0`, given the fact that EMR will continue to have 3.0 support in old versions that users can choose.

Reply:
ok I backed this out and just added "Spark 3.0" to the reference above